code/modules/mob/living/simple_animal/hostile/terror_spiders/ghost.dm

```diff
@@ -0,0 +1,44 @@
+
+/mob/living/simple_animal/hostile/poison/terror_spider/Topic(href, href_list)
+	if(href_list["activate"])
+		var/mob/dead/observer/ghost = usr
+		if(istype(ghost))
+			humanize_spider(ghost)
+
+/mob/living/simple_animal/hostile/poison/terror_spider/attack_ghost(mob/user)
+	humanize_spider(user)
+
+/mob/living/simple_animal/hostile/poison/terror_spider/proc/humanize_spider(mob/user)
+	if(key)//Someone is in it
```

Comment:
I did have that in a previous build. It was annoying because everytime someone clicked a spider to follow it, they'd get the error.